### DISCUSSION TITLE
UI indication that a window is currently in fullscreen mode

### DISCUSSION THREAD

**Discussion by samuela** [nikitabobko/AeroSpace#1017] [feature-ideas]
Having the ability to temporarily break the layout and make a single window fullscreen is useful. However, looking at a fullscreen window, there is no indication AFAICT that I am looking at a window in fullscreen mode as opposed to a workspace with only a single window.

Is there a way to give the user some indication that they are in fullscreen mode? Perhaps something like service mode where the menu bar goes from `1` to `[s] 1`?

**Comment by Hylian** [comment]
Being able to query fullscreen status via something like `aerospace list-windows` or `aerospace list-windows --focused --fullscreen` would be handy for displaying the status in Sketchybar.
